tests/integration/sugar/test_transaction.py

```diff
@@ -223,8 +220,8 @@ async def test_calculate_account_delete_fee(self, client):
             account_delete, WALLET, client
         )
 
-        # THEN we expect the calculated fee to be 5000000 drops (5 XRP)
-        expected_fee = "5000000"
+        # THEN we expect the calculated fee to be 50000000 drops (5 XRP)
+        expected_fee = "50000000"
```

Comment:
Is this change expected? Changing fee from 5 to 50 XRP? If so, then the comment needs to be updated to `(50 XRP)`

Reply:
Oh I'll look into that some more - I didn't actually calculate how many XRP it was, I assumed it was an error.